src/components/DropzoneArea.js

```diff
@@ -49,8 +49,10 @@ class DropzoneArea extends React.PureComponent {
         const {initialFiles} = this.props;
         try {
             const fileObjs = await Promise.all(
-                initialFiles.map(async(url) => {
-                    const file = await createFileFromUrl(url);
+                initialFiles.map(async(file) => {
+                    if (typeof file === 'string' ) {
+                        file = await createFileFromUrl(file);
```

Comment:
Here you should avoid overwriting the `file` parameter.
`file` param name should be `initialFile` and you'd need to add a new temporary variable `file`.